care_scribe/tasks/scribe.py

```
@@ -501,33 +580,40 @@ def generate_response(retry=0):
+
+        logger.info(f"Scribe[{form.external_id}] status: completed AI form processing")
+
+        # Update the user and facility quotas
+        if not is_benchmark:
+            user_quota.calculate_used()
+            facility_quota.calculate_used()
 
     except Exception as e:
-        # Log the error or handle it as needed
-        logger.error(f"AI form fill processing failed at line {e.__traceback__.tb_lineno}: {e}")
+        logger.error(f"Scribe[{form.external_id}] status: error occurred while processing form at line {e.__traceback__.tb_lineno}: {e}")
```

Ruff (0.14.8)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)